creator-node/src/resizeImage.js

```diff
@@ -125,7 +122,7 @@ module.exports = async (job) => {
     })
   )
 
-  // Add all the images to IPFS including the original
+  // Compute multihash/CID of all the images, including the original
```

Comment:
nit - can just use CID

Reply:
lol i see the functions are called `generateImageMultihashes` its fine haha, we should prob standardize it but doesnt have to be in this PR

Reply:
I read something somewhere saying we want to use the term "multihash" instead of "CID" going forward, but are you saying all of these references should be called CID? I can definitely change it to reflect that in the next PR

Reply:
its the opposite - we should actually standardize to CID, but yeah subsequent PR makes sense